job-dsl-plugin/src/main/groovy/javaposse/jobdsl/plugin/ExecuteDslScripts.java

```diff
@@ -191,18 +194,38 @@ public String getTargets() {
         return targets;
     }
 
+    @DataBoundSetter
+    public void setTargets(String targets) {
+        this.targets = StringUtils.isBlank(targets) ? null : targets;
+    }
+
     public String getScriptText() {
         return scriptText;
     }
 
+    @DataBoundSetter
+    public void setScriptText(String scriptText) {
+        this.scriptText = StringUtils.isBlank(scriptText) ? null : scriptText;
```

Comment:
Consider using [hudson.Util.fixEmpty(string)](http://javadoc.jenkins-ci.org/hudson/Util.html#fixEmpty-java.lang.String-) for the exact same behavior.